Code/Tools/ProjectManager/Platform/Windows/ProjectUtils_windows.cpp

```diff
@@ -77,7 +77,7 @@ namespace O3DE::ProjectManager
             if (vsWhereFile.exists() && vsWhereFile.isFile())
             {
                 QStringList vsWhereBaseArguments = QStringList{"-version",
-                                                               "16.9.2",
+                                                               "[16.9.2,17)",
```

Comment:
This change too - not sure if we want to allow higher versions in dev or restrict to VS 2019